app/src/main/java/org/mozilla/tv/firefox/telemetry/SentryIntegration.kt

```diff
@@ -60,7 +61,24 @@ object SentryIntegration {
         }
     }
 
+    /**
+     * Sends the given [exception] to the Sentry servers without crashing the app.
+     *
+     * @see [Sentry.capture]
+     */
     fun capture(exception: Exception) {
         Sentry.capture(exception)
     }
+
+    /**
+     * Sends the given [exception] to the Sentry servers without crashing the app
+     * and logs its message at the error level.
+     *
+     * @see [Sentry.capture]
+     */
+    fun captureAndLogError(tag: String, exception: Exception) {
+        // Note: instead, by adding the Log4j dependency, we may be able to get Sentry to log captures to logcat automatically.
+        Log.e(tag, exception.message)
```

Comment:
Sure, seems reasonable: I think we should send an email out encouraging everyone to use the a-c logger instead of Android's with the reasons why and a suggested pattern. I did:
```kotlin
private val logger = Logger("tag")

public class Whatever...
```

@Baron-Severin Would you like to take that on?

Reply:
Sure, works for me.